frontend/src/services/graphQLDevice.ts

```diff
@@ -107,6 +161,43 @@ export async function graphQLFetchDevice(id: string) {
   )
 }
 
+export async function graphQLGetMoreLogs(id: string, from: number, maxDate: string) {
+  return await graphQLRequest(
+    `
+        query($ids: [String!], $from: Int, $maxDate: DateTime ) {
+          login {
+            id
+            devices(id: $ids) ${LOG_SELECT_FOR_DEVICE}
```

Comment:
We will want the single device query here as it's much more performant. But again let's wait for account linking as it's changed how to query a bit.

Reply:
You should now be able to reference the single device query from account linking.